app/src/routes/routesMap.js

```diff
@@ -101,6 +101,7 @@ export default {
     thunk: (dispatch) => dispatch(fetchMembersAction()),
   },
   PROJECT_SETTINGS_PAGE: '/:projectId/settings',
```

Comment:
Now we have PROJECT_SETTINGS_TAB_PAGE route, so all attempts to get on PROJECT_SETTINGS_PAGE should be redirected to '/:projectId/settings/general'. Please add it.